### ISSUE TITLE
[Bug] - Random Pop up warning " Installed filters will continue to work even if you exit. Continue ?"

### ISSUE THREAD

**Issue by Lordvader81** [henrypp/simplewall#1598]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.7.3

### Windows version

11

### Steps to reproduce

Random ??
There doesn't seem to be any specific reason or Time when or why it happens.. 

### Actual behavior

Notification box randomly pops up in random intervals As if I was manually closing the application from the tray. But im not?

![image](https://github.com/henrypp/simplewall/assets/120199915/ccf2ff38-7ffe-4a57-9636-814b1e4bd572)

**Comment by henrypp** [comment]
Where it is random? If you "manually closing the application from the tray" there was no random at all. And where is the problem to check "Do not ask again" to avoid this LOGICAL message in next time???

**Comment by Lordvader81** [comment]
Yes, I understand that...🤦‍♂️
It's random bc I'm not clicking the app from the tray and closing it ....
The warning pops up on its own with no use interaction..
And guess what .... Even if I check ✔️ the box saying "Do not ask again" it
still happens again!!

**Comment by henrypp** [comment]
with no interaction? you have friend who jokink for you?

**Comment by Lordvader81** [comment]
I have gathered some additional information about this issue. It appears to be related to the screen saver and monitor power-off cycle.

 

I've noticed that it's not as random as I initially believed. The problem occurs when I return to my computer after approximately 30 to 45 minutes. My monitors are configured to power off or enter a low-power state automatically after 30 minutes. The computer is currently set to high-power mode and is not configured to hibernate or sleep under the mentioned monitor conditions. When I return and wake up the monitors, I encounter the warning pop-up box we discussed.
